source/views/dictionary/list.js

```diff
@@ -130,6 +130,7 @@ export class DictionaryView extends React.Component {
         <SearchBar
           getRef={ref => (this.searchBar = ref)}
           onChangeText={this.performSearch}
+          onSearchButtonPress={() => this.searchBar.unFocus()}
```

Comment:
Sure, this is fine. Could you add a comment above it, like

```js
onChangeText=…
// if we don't use the arrow function here, `this.searchBar` is null...
onSearchButtonPress=…
```

Otherwise I'll probably keep trying to change it in the future.